Kudu.Contracts/IEnvironment.cs

```diff
@@ -13,6 +13,11 @@ public interface IEnvironment
         string SSHKeyPath { get; }
         string TempPath { get; }
         string ScriptPath { get; }
+
+        /// <summary>
+        /// Path to Kudu SCM site binary folder
```

Comment:
good idea. updated comment to be

/// <summary>
/// <para>Path to Kudu SCM site binary folder</para>
/// <para>e.g E:\kudu\Kudu.Services.Web\bin</para>
/// </summary>

and FYI this "binPath" is not new. in Environment.cs, it is used it to create node module path etc ..